business/services.go

```diff
@@ -294,6 +319,7 @@ func (in *SvcService) buildRegistryServices(rSvcs []*kubernetes.RegistryService,
 			HealthAnnotations: map[string]string{},
 			Labels:            item.Attributes.Labels,
 			Selector:          item.Attributes.LabelSelectors,
+			ExportTo:          item.Attributes.ExportTo,
```

Comment:
I think that the "ExportTo" is only required for the "AllNamespaces" flag (used only for validations). Am I right ?
I mean, the buildServiceList() complements the Kubernertes services with the Istio Registry ones, if we may want to use this field in other use case, perhaps we'd need to combine both in that point.

No action, from my side, just thinking loud on this.

Reply:
I think this line is odd now and can be removed, as Exported Services validations will be done directly in FilterByRegistryService in filters.